test/test_utils.py

```diff
@@ -243,3 +243,19 @@ def test_not_same(self):
         self.assertFalse(util.compare_barcodes('727361379704', '1727361379704'))
         self.assertFalse(util.compare_barcodes('0727361379704', None))
         self.assertFalse(util.compare_barcodes(None, '0727361379704'))
+
+
+class MbidValidateTest(unittest.TestCase):
+
+    def test_ok(self):
+        self.assertEqual(util.mbid_validate('2944824d-4c26-476f-a981-be849081942f'), True)
```

Comment:
You can directly use `assertTrue` / `assertFalse` here

Reply:
Done in  5af33683921cbf5e12ca795a01f893cf26414119 (rebased)